### PR TITLE
Updated api docs with the new `localCost` field.

### PR DESCRIPTION
**Pull request by JulietaMarcos**
- Added description of the new `localCost` field
- Deprecated the old `cost` field
- Fixed issue when displaying `sources` inside segment templates

The new field isn't yet deployed. @MarianoTucat can you merge this after the next deploy of production code, if @nighthawk approves these changes?

### CODE REVIEW

File: docs/specs/tripgo.swagger.yaml [hunk 1]
```diff
@@ -2801,7 +2831,8 @@ definitions:
           $ref: '#/definitions/Location'
         cost:
           type: number
-          description: Cost of this segment in local currency
+          description: |
+             **Deprecated.** Cost of this segment in local currency
```
Comment (nighthawk) [inline docs/specs/tripgo.swagger.yaml]: I believe that some Swagger tools support a `deprecated` boolean. Please add one.

File: docs/specs/tripgo.swagger.yaml [hunk 2]
```diff
@@ -2107,6 +2112,33 @@ definitions:
       #     - label: Sat - Sun
       #       price: 22
 
+  LocalCost:
+    description: Information about the money cost of this segment in local currency. Missing when price doesn't apply or is unknown.
```
Comment (nighthawk) [inline docs/specs/tripgo.swagger.yaml]: Please add an example behind doesn't apply, such as `(e.g. walking)`

Comment (JulietaMarcos) [reply docs/specs/tripgo.swagger.yaml]: Done

File: docs/specs/tripgo.swagger.yaml (hunk 1 above)
Comment (JulietaMarcos) [reply docs/specs/tripgo.swagger.yaml]: I added it, but it seems that `deprecated` on properties is only possible in OAS 3, and ReDocs doesn't support it. So I left it in the description too. Did the same thing for `disclaimer`.

**Pull request merged**